desktop/modal/recovery.atlas.php

```diff
@@ -39,7 +39,7 @@
     $('.textAtlas').text('{{Vous pouvez démarrer la restauration de votre box Jeedom Atlas.}}');
   } else if (typeDemande == 'usb') {
     startDemande = 'startUSB'
-    $('.textAtlas').text('{{Veuillez brancher une clé USB de plus de 10Go.}}');
+    $('.textAtlas').text('{{Veuillez brancher une clé USB > à 10Go (port USB noir de droite).}}');
```

Comment:
It's a good practice to avoid hardcoding text directly in your code. Instead, consider using constants or a separate configuration file to store these strings. This will make it easier to manage and update the text. [medium]